src/01-general/document-templates.apib

```diff
@@ -0,0 +1,49 @@
+## Document Templates [/documentTemplates]
+
+### documentTemplates.list [GET /documentTemplates.list]
```

Comment:
NTH: It looks like all endpoints described in https://developer.teamleader.eu are `POST` (which makes sense as we're passing bodies)
I'd keep this consistent and also mark this as `POST`